src/Navigation/SidePanel/SidePanel.stories.tsx

```diff
@@ -28,15 +28,20 @@ export default {
 }
 
 const DefaultStoryTemplate = (args: SidePanelProps) => {
-  const [current, setCurrent] = React.useState({})
+  const [items] = React.useState(fixtures[args.variant || 'default'])
+  const [current, setCurrent] = React.useState({
```

Comment:
Would it look nice if you import useState with `import { useState } from 'react'` then type **useState** directly without writing **React.useState**? Is it like this everywhere ( if yes ignore me ) .. The reason why I'm bringing this up is because in the other repo we use `import { useState } from 'react'`

Reply:
This is a preference choice and I found it less text is more, easier for the brain also you know where it comes, so for juniors should become intuitive - the reasoning behind. I'm not changing